.github/workflows/fuzzers.yml

```diff
@@ -39,10 +39,20 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+
+    - name: Clear unnecessary files
+      run: |
+        sudo swapoff -a
+        sudo rm -f /swapfile
```

Comment:
Hmm...Any idea how much this saves? Not sure if it is a good idea to get rid of swap entirely (maybe it is necessary given our space issues).

Reply:
By "I don't know" I really mean that I have no idea. Not trying to politely imply that I do know this is bad.

Reply:
Ok trying, will comment back.